src/main/java/com/skhu/gdgocteambuildingproject/auth/controller/AuthController.java

```diff
@@ -43,4 +42,10 @@ public ResponseEntity<Void> logout(@RequestBody RefreshTokenRequestDto dto) {
         authService.logout(dto);
         return ResponseEntity.noContent().build();
     }
+
+    @DeleteMapping("/delete")
+    public ResponseEntity<Void> delete(@AuthenticationPrincipal UserPrincipal userPrincipal) {
+        authService.delete(userPrincipal.getUser().getId());
+        return ResponseEntity.noContent().build();
+    }
```

Comment:
```suggestion
    @DeleteMapping
    public ResponseEntity<Void> delete(@AuthenticationPrincipal UserPrincipal userPrincipal) {
        authService.delete(userPrincipal.getUser().getId());
        return ResponseEntity.noContent().build();
    }
```

컨트롤 URI를 사용하지 않아도 API의 역할을 이해할 수 있을 것 같아요 😄